pom.xml

```diff
@@ -152,7 +152,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <performance.ei.version>0.1.0-SNAPSHOT</performance.ei.version>
-        <performance.common.version>0.2.2-SNAPSHOT</performance.common.version>
+        <performance.common.version>0.3.1-SNAPSHOT</performance.common.version>
```

Comment:
Why are we using a snapshot version here? Can we not use the latest released version?